### PR TITLE
Modified README project description

### PR DESCRIPTION
**Pull request by rhahn07**
Generated database with three tables for restaurants, reviewers and reviews.  Any comments will be most appreciated.

### CODE REVIEW

**Review by cstung21 — COMMENTED** [review]
Updated description looks good

File: data/generateReviewData.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+var review = function(restaurantId) {
+  var obj = {
+	restaurantId: restaurantId,
+	reviewerId: Math.ceil(Math.random() * Math.floor(25)),
+  stars: Math.floor(Math.random() * Math.floor(5)),
+  reviewDate: faker.date.recent(),
+  checkIns: Math.floor(Math.random() * Math.floor(5)),
+  textBody: faker.lorem.paragraphs(Math.ceil(Math.random() * Math.floor(4)))
+  };
```
Comment (KiltedDrummer) [inline data/generateReviewData.js]: unnecessary Math.floor for integers.

File: data/generateReviewData.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+var faker = require('faker');var restaurantData = require('./fecData.js')
+var restaurantData = require('./fecData.js')
```
Comment (KiltedDrummer) [inline data/generateReviewData.js]: var restaurantData = require('./fecData.js')
is repeated

File: package.json [hunk 3]
```diff
@@ -7,11 +7,16 @@
...
-  "scripts": {
-  },
+  "scripts": {},
   "dependencies": {
+    "express": "^4.16.3",
+    "faker": "^4.1.0"
   },
   "devDependencies": {
-    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
```
Comment (KiltedDrummer) [inline package.json]: if you can get airbnb linter added, it's recommended from HR. is more strict, but build really good practices.

**Review by KiltedDrummer — COMMENTED** [review]
overall looks good, I don't think most of your Math.floor()'s are needed as they are called on an integer.
Math.floor(5) === 5, so could clean that up.

also, I noticed you are using 'var' for everything, I'm not sure if it's required for FEC, but i think they wanted us using 'let' and 'const' for ES6 best practices. might want to check with Toby on that though.

EDIT: I pinged Toby, you may want to talk with him if you are unsure, but he was saying best practices is not to use partial ES6, and if you are using other ES6 conventions to refactor your var's.
I did not read to see if there are other ES6 conventions, I was going to leave that up to you / you and Toby

File: package.json (hunk 3 above)
Comment (rhahn07) [reply package.json]: will do. thanks, nick!

File: data/generateReviewData.js (hunk 1 above)
Comment (rhahn07) [reply data/generateReviewData.js]: aha! thanks for pointing that out

**Pull request merged**